src/core/stack_switching/esbuild.config.mjs

```diff
@@ -4,10 +4,38 @@
  */
 
 import { build } from "esbuild";
+import { readFileSync } from "node:fs";
+import loadWabt from "../../js/node_modules/wabt/index.js";
```

Comment:
Can we avoid the hardcoded assumption as to where node_modules are? It doesn't work with a non relative import?

Reply:
Problem is that this directory isn't under `node_modules`. Most likely there is a better way, like moving this script into `src/js` or something. But I'm not sure what the best approach is.

Reply:
I would like to leave fixing this to a followup.